cmd/ocsp-updater/main_test.go

```diff
@@ -773,81 +773,92 @@ func TestMissingLogs(t *testing.T) {
 	}
 
 	testCases := []struct {
-		Logs                []*ctLog
-		GivenIDs            []string
-		ExpectedMissingLogs []*ctLog
+		description         string
+		logs                []*ctLog
+		givenIDs            []string
+		expectedMissingLogs []*ctLog
 	}{
-		// With `nil` logs, no log IDs are ever missing
 		{
-			Logs:                nil,
-			GivenIDs:            []string{testLogAID, testLogBID},
-			ExpectedMissingLogs: []*ctLog{},
+			"With `nil` logs, no log IDs are ever missing",
+			nil,
```

Comment:
Thanks for the link to the Go stdlib tests, that's informative! Our general style has been to always use field names. This is largely informed by either `go vet` or `go lint`, which complains if they are not present. We used to run both and enforce them in our CI, but we dropped one of them since it was specifically not intended for CI. In general, I'd prefer to leave the field names in without a strong reason to refactor in the other direction.